package.json

```diff
@@ -64,7 +64,7 @@
   },
   "jest": {
     "transformIgnorePatterns": [
-      "/node_modules/(?!@edma\/(design-tokens|blocks))"
+      "/node_modules/(?!@edma/(design-tokens|blocks))"
```

Comment:
@lheld-pnmac @abbyjb Could you run this locally on your macs to see if we need to update this regex to be compatible with Mac + Windows.

Reply:
The fact that it passed the PR gate (which is running on a ubuntu box) makes me think we're okay.